db/migrate/20190315225354_delete_all_pending_delayed_jobs.rb

```diff
@@ -0,0 +1,5 @@
+class DeleteAllPendingDelayedJobs < ActiveRecord::Migration
+  def up
+    Delayed::Job.delete_all
```

Comment:
I'm not sure this is a good idea. There may be pending jobs that are still important. I think we should delete specific job types that are now obsolete.

Reply:
ok, I have changed it to only delete obsolete jobs related to account invoices. Please have a look.
ping @kristinalim are you ok with this?

Reply:
@luisramos0 Yes, I'm okay with this.

Reply:
In relation to this comment, I did quick check in recent commits and spotted that `SubscriptionConfirmJob` and `SubscriptionPlacementJob` have been renamed. Pending jobs of this type would not have a corresponding job class in v2. I created #3643 to address this.

Reply:
thank you!